pennylane/devices/tests/test_compare_default_qubit.py

```diff
@@ -0,0 +1,211 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests that a device gives the same output of the default device."""
+import random
+
+import pytest
+from flaky import flaky
+
+import pennylane as qml
+
+# pylint: disable=no-self-use
+from pennylane import numpy as np
+from pennylane.templates.layers import RandomLayers
```

Comment:
@trbromley adding `from pennylane import numpy as np` was a suggestion from my part previously (see https://github.com/PennyLaneAI/pennylane/pull/897#discussion_r521781150). Maybe we could leave a comment here to make it explicit, e.g., (`# Importing NumPy from PennyLane to test the environment demonstrations use` or something similar).

Reply:
Thanks @antalszava! Sure I'm potentially happy to have `from pennylane import numpy as np` (although would be good to add a `no-member` to the pylint disable).

What do you mean about environment demonstrations? As in the demos [here](https://pennylane.ai/qml/demonstrations.html)?

Reply:
Yep, some demos (e.g., [Variational classifier](https://pennylane.ai/qml/demos/tutorial_variational_classifier.html), [Quanvolutional Neural Networks](https://pennylane.ai/qml/demos/tutorial_quanvolution.html)) have this import and use arrays to represent data. When processing user questions it turned out that the difference between using a `pennylane.numpy.tensor` instead of the original `np.array` can result in errors due to bugs.

Perhaps the word `environment` is not the best to use here, could also go with `# Importing NumPy from PennyLane to use similar imports as in QML demonstrations` or something better.